### ISSUE TITLE
+X to Level of All Gems Affixes Not Increasing Gem Level

### ISSUE THREAD

**Issue by UsernameNAN** [PathOfBuildingCommunity/PathOfBuilding#5992]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the calculation is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the value from the calculation in-game?

"+X Level to All Y Skill Gems" modifier increments the level of all gems of Y type by X.

### What is the value from the calculation in Path of Building?

These modifiers are not incrementing skill gem levels at all. This includes both weapon "spell skill gems" and amulet "skill gems" mods, both rares and uniques.

### How to reproduce the issue

1. Select Arc as main skill
2. Create amulet with "+1 to Level of all Skill Gems" modifier.
3. Note that adding or removing this mod does not effect damage.

### Character build code

```shell
eNqtG1tv4jr6efgVEdJKuzrbQri1U9FzRIG2SKVloTOz-zQyiQGfOjaTOG05R_Pf97OdQGBwmpD0oQ3xd_d3NW73j3ePWq_YDwhn11X7vF61MHO4S9jyuvrl-fbssvrH75XuBInV0-ImJFSu_F751FXPFsWvmF5XPwOaQP4Si68xqeZ3eLdGTKwwZ2P0J_fvuHtdfeQMV605Yi4R8SeHoiB4RB6-rn4jwllVLRQ4mLn93fsIcIV85AjsP0i2vVDwMXdhdYFoAMseImzGnRcs7nwerkGdqvVK8JsGGo0nT9PnhEyEJWUCnT51JxRtsD8TSFgB_Lqu9sA0aInviQBSiIZAp2mfNzqNdrWWinET-oEYIA8es2LO1hi7W2D7vNW4bF_Y0W8T0sTHw8UCO4K84r5PRH-FmLNjaWSWF3YcUkHWlGA_IaER4_4X4na9bgJ-5gLRwWS2hW21zjud-oeqazy-2xkjB_Cq1Q0F657AReKOlowIfCLyhJOAswL6JVGNKvZDSiEyM8FOcYD9VyTIvlhm2tybE3aS9caIoT4Pdnt0mQY5wT7EvdhDqH-AMMMOh1SRRLH3YycDx-N0jKwfyAJnh8ylVYSQV5rT9BjOssLlJnyaQFNIl9kgZzykGSFFImUZHXWA3zNAjVjCky_M8fUjCdhsmpm-chmFHyuhcsHwfpLw8frlefOi1bxsNZodYyKerDYBcRAdo3fihR7k5Wf0glmCjLkePZDlSjBIKibki7pRs1vi4xPQ-py6p6CtEA9OwJORk8kUUJqdKwk9Yk62gPzCfJVlkxU91dYLPIVAkc3DnOLMODs2UcBlKbia2RKziOMmm0oPGDurO-ixpkjgbPl5C9WxL1JtK4Ez2VYCHrFtCv19jBxmkojHzWR3zlMrXk5LDRn2l5vZimDq5oOOJeujdYZEKA2dxM5k8H12uZwmiZrTJN-Q72YrF3llekVBMuPa6TEfgWdzTQwtJiC4-KD3rZubd_6nbN1pPrSe7_HQz7jhGjiTAnGx0GPLFLuhk6063VCYurJKD1JRmgujJwRyXgbcXeJcTPJjzML1GjKF3PKseLLSQUNNEq3IWScD9BP4a6awlTUxO4MddGYG2zqfncsBSnZdZKHOocwOPDOL7TQ8hozgQapXY_WYJ-qwcXNghso0ECnAjIPZhL-B5Ct5OBLkg4aGZtdDGkXxMftrk5n-HngmBkPmhr4Mhcw8DjGOsXkmHmTLIBgggSw3aoO_Ip8gJhrq4CbAyHdWD7D1t4jSOSSA62ryrfx0gGjHe9utqdMp-TTy1twX6mUfUSdQ3EdsHQqLqZMljwTO93m4WMhTImAgfHX0Nby9HfafR1-HkcBJlOCFUPqdhd5cnobovzu3mmGVNa0gnAf68br6leC3mcQaYIEIBSM6nFK0DrC7Pb0i8KhgZmAZR6RQU1DQ58bHS8dp7QDMlIbv2BegL5RbxyfYKNd2_QOhNENZiuUGm6jJMxszIZ3t-ygQulUwWEodlpmpyDMrozpyMQUXfAZRI-do9QNLiM0aB1B_yYI4MjbSt_wZoDVUil0cB8LK2aTsd1SzzDTUIZiJgF40I-uDLBN2tJpiVXWIZrSqXjWjD7CDjLrrRTPytvvkTB3lHqeyhUqh9MiZcnIImh6hsr4Yd3ZI8RbETPBJrLCvi5aR0hhyVAySGjg-mYfCHMYJiBRbqaHYYCG5ZkbVI59BB7mWkon2ZiCDQZMwZlJ6djAmsjRU3WsY7Rd1LilbEHXmBvPr1RQjxMOJQf9oOSVIVP7tvXLi6u7VEC4HYGkJA_r04mRUJ16czGGPXpziLUXBi3G_o1Uz-heY3YjYHKEi_DArERlUxSjI2CpGYXrYSOxwp-kthMpfA7zAYOjUBLaFSdlfEbIBFDmRsrcZSSmxjgfCTrVctHQqP6ppboraQaNzyjQf1iAfEIJadJ_S7WSjtJ2b7jGi8lsrTosR_OU8tpCeXASIuQN52FNQUXlWFK6BWCzZ07F2dbelh1S7tXiQ6KreLbACmDDusBfcbGAivJXV6eA7M6QMGzX3Qo83SZyYnYsXKKTy_X9CJBOLnLISbx_0t-2M-972SB9IrfibTIqaomwlA6g1Dw96pUdFREyyi-2lrRMJpNS01bvopfoqvbeTWmms5CbMoaGLRywalkEaAo5C0VxKJi8PyPPrrQEP6GzZfOqCNBHwHeVzRBsxSnSlwK7uLdvxshq-RqqaOlXrh1ZtpHpAZaaqtcSe_DyGUcuF8bI2EqB4TWpfU0LAUxJX2fhAbPAQpvdJTnzyHE3zq2kbaTrKhrXYiMoztEPIx2cf43jfFbod2Rw-JK83SEnr0k6BwP4mavKkIRmMo_DQbrVaF1VLALnE1YqGHd2akOjNaOe-TB-0bVdCrIOrWu3t7e18jcSKL_A7ofjc4V5tDUgg05ky45kkW-vBz81yLP_0tHoxpa6-TRHUIlVBdqWmVE4-PHJoJeWafBl_UKrLgTca1mfCl_r8xbn3P6WrfIo8uRl5L4w-AwK6-ypxxrsgAf8bHzR01TZGJpXPcSSFAdZfMX7DaM2Zei19Uo_tAJh07ymC0W9zZU1702HlEWSUAJUbGmJrAtJSq-eFFItK30cLgd0rS0pSmfh4Qd6vrL99xJb4qn7e_qm9MvYnpY69hZMXSvY_zMJFkoL9c8QE9DJkKRX-vF3eg1Ufoti9suoVxWSKf1xZFxcVmEIpcSDrX1l25W-BlsEVNEXQXjr43x60Az8Tsv6zdXnW7vzrHzJ4fYwgv1myY7DU0TnW5daSw3blt3bbEtxKClf5zZavFHOLLywE2UHnCBlRykvG3J1KZpZiCRt23q5GRo8CxgjR-BAiPjFSYbzdU5m1TBu_n84oh_wGGJM5eLUKLHWGA_v3Kqu4XNIxWEtH0EwsOwfKDYeqZfXmmyBAYDQVS1Y7DwFMxSF-Iwe-avus5glaWrM3tD5k3TqFUAH1IxqNEmgcV6hTgkKdEhQ6icYxhQp7Z7MEizTK0iaPv91jaM9FIXscCbZOYQkauSkUZtksy_7tEpzBLiE87BLkaOX0hLJSTKNoRLZKMGD-mlFaim0W1b9RMITzCeBuLH1KV8QHdW9RhMLxwG4XptApa1vtsnJMftcsbsdW4b1sl-NUeZTXg8kOwT41EzZLSCjNgkHZKohvl2P_kytlbqdt5cY4pYsqaNV20Vxd0ra0ykoupfXVJ49QOfCmhC1zbeHx1NQoZw9KqxRlDHyt03TKa_ziNb6kZqNdWJDiJa6TNxkUTR6_MtRnPvHZqjr_U_d4OFuQZXz3yMErTl3sR6Qxw94m-k-q-GbORfKO7zF4eR8kvkwTIzU_wIldbHv956LRufyYTXytMEZrp6NEVzE5pep0LKlTOuL2q-QY4bJ5-YF4-9c_E5o1Prfibz4i03drh_8Q-H9yTuSK
```

### Screenshots

![image](https://user-images.githubusercontent.com/130267605/230754101-f8a56542-95aa-44ec-a04b-2436a2d463bc.png)
![image](https://user-images.githubusercontent.com/130267605/230754111-c5973e88-307f-4e37-af38-b9b174677143.png)

**Comment by QuickStick123** [comment]
Duplicate of https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/597 and already fixed in dev by https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/5982

**Comment by UsernameNAN** [comment]
> Duplicate of #5976 and already fixed in dev by #5982

Ack! I didn't check in closed. My apologies, not used to github.

**Comment by QuickStick123** [comment]
All good don't be too bothered it is better that we hear about an issue then don't.
